airgun/widgets.py

```diff
@@ -949,7 +949,7 @@ def add(self, value):
         :param value: dict with format {'name': str, 'value': str}
         """
         self.add_new_value.click()
-        new_row = self.__getitem__(-1)
+        new_row = self.__getitem__(0)
```

Comment:
new rows are now adding from the top, not from the bottom

Reply:
Can you test this that it is not breaking other places this code has been used